workflows/amplicon/qiime2/qiime2-I-import/QIIME2-Ia-multiplexed-data-single-end.ga

```diff
@@ -80,7 +80,7 @@
             "input_connections": {},
             "inputs": [
                 {
-                    "description": "A tab separated metadata file",
+                    "description": "Metadata",
```

Comment:
This is strange, why the autoupdate changes the description???

Reply:
Indeed. Seems to be a bug?

Reply:
It just overwrites this with the value of `name`.

Reply:
Also it seemed to happen only for the workflows in this PR. 

Import workflows Ic and Id seem to be missing.

Reply:
Yes if Ic and Id has no updated tools they are not updated so not in this PR but you need to manually change their versions. This is a flow in the autoupdate and this is absent from the linting so we need to be careful about it.

Reply:
Seems that the description attribute inside of the inputs list (of the input) is copied from the annotation attribute .. fixed in https://github.com/galaxyproject/iwc/pull/422/commits/6027fef3cbdc870c0bbb8d390cb760d366f7c72c